src/main/java/in/koreatech/koin/domain/user/UserToken.java

```diff
@@ -0,0 +1,31 @@
+package in.koreatech.koin.domain.user;
+
+import java.util.concurrent.TimeUnit;
+import lombok.Getter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.redis.core.RedisHash;
+import org.springframework.data.redis.core.TimeToLive;
+
+@Getter
+@RedisHash("refreshToken")
+public class UserToken {
+
+    @Id
+    private Long id;
+
+    private final String refreshToken;
+
+    @TimeToLive(unit = TimeUnit.DAYS)
+    private final Long expiration;
+
+    private UserToken(Long id, String refreshToken, Long expiration) {
+        this.id = id;
+        this.refreshToken = refreshToken;
+        this.expiration = expiration;
+
+    }
+
+    public static UserToken create(Long userId, String refreshToken) {
+        return new UserToken(userId, refreshToken, 3L);
```

Comment:
기존에는 
- 시간 관련
  - AccessToken 만료기한: 3D (안드로이드까지 리프레시 기능 추가시 2H로 변경 예정.)
  - RefreshToken 만료기한, 레디스  TTL: 14D

로 되어있는데 통일이 필요하지 않을까요?
```suggestion
        return new UserToken(userId, refreshToken, 14L);
```

Reply:
14일로 수정하겠습니다~!